service/lib/dinstaller/storage/proposal.rb

```diff
@@ -19,36 +19,38 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "y2storage/storage_manager"
-require "y2storage/guided_proposal"
-require "y2storage/proposal_settings"
+require "y2storage"
 require "y2storage/dialogs/guided_setup/helpers/disk"
 require "dinstaller/with_progress"
+require "dinstaller/storage/actions"
+require "dinstaller/storage/proposal_settings"
+require "dinstaller/storage/proposal_settings_converter"
+require "dinstaller/storage/volume_converter"
 
 module DInstaller
   module Storage
     # Backend class to calculate a storage proposal
```

Comment:
Perhaps adding an @example here could help to make a better idea of how this class works.